onebusaway-android/src/google/java/org/onebusaway/android/map/googlemapsv2/BaseMapFragment.java

```diff
@@ -738,6 +738,7 @@ public void onLocationChanged(Location l) {
         if (firstFix) {
             // If its our first location (across all BaseMapFragments), move the camera to it
             setMyLocation(true, false);
+            onFocusChanged(null,null,l);
```

Comment:
Just realized that this change is still in this branch.  We changed this model in a previous PR to not using this callback on first GPS fix, so this branch should be rebased on develop to take advantage of the new structure.  You can wait to do this until we merge the other PR for scripting Amazon classes so all the debasing is done in one step.